modules/packages/LinearAlgebra.chpl

```diff
@@ -390,8 +390,10 @@ proc Matrix(const Arrays: ?t ...?n, type eltType) where isArrayType(t) && t.rank
 
   var M: [{dim1, dim2}] eltType;
 
-  for i in dim1 do
+  for i in dim1 do {
```

Comment:
Sorry if my previous comment wasn't clear - I think it's OK to use a `forall` here, optimizing for the creation of large arrays. We can come back and optimize this further in the future.

Reply:
You were clear. I meant to change this to forall and mistakenly kept it as for. It should be forall now.